bayes_people_tracker:
    prune_named: true
    filter_type: "UKF"                                         # The Kalman filter type: EKF = Extended Kalman Filter, UKF = Uncented Kalman Filter
    cv_noise_params:                                           # The noise for the constant velocity prediction model
        x: 1.2
        y: 1.2
    detectors:                                                 # Add detectors under this namespace
        openpose_tracker:                                       # Name of detector (used internally to identify them). Has to be unique.
            topic: "/person_rec_lw/poses"  
            cartesian_noise_params:                            # The noise for the cartesian observation model
                x: 0.2
                y: 0.2
            observation_model: "CARTESIAN"
            matching_algorithm: "NNJPDA"                     # The algorthim to match different detections. NN = Nearest Neighbour, NNJPDA = NN Joint Probability Data Association
        tracking_actuator:
            people_topic: "/tracking_act/people"
            cartesian_noise_params:
                x: 0.2
                y: 0.2
            observation_model: "CARTESIAN"
            matching_alogrithm: "NNJPDA"
